Formula/wso2ei-6.4.0.rb

```diff
@@ -0,0 +1,35 @@
+class Wso2ei640 < Formula
```

Comment:
This needs to be renamed so it doesn't include the version.

This needs to be done with all of the open PR.

Reply:
Isn't this preventing us from using the package name as `wso2ei-6.4.0`?
We have a requirement to install the  products as `brew install wso2ei-#{version}`

Reply:
The formula should able to be installed as `brew install wso2ei`.

If having them versioned is important I'd suggest [creating your own tap.](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md)